packages/services/src/infrastructure/rate-limiter/rate-limiter.service.ts

```diff
@@ -14,11 +14,17 @@ import { leatherApiLimiter } from './leather-rate-limiter';
 if (!AbortSignal.prototype.throwIfAborted) {
   AbortSignal.prototype.throwIfAborted = function throwIfAborted() {
     if (this.aborted) {
-      throw new Error('AbortError');
+      throwAbortError();
     }
   };
 }
```

Comment:
If this is only for RN, shouldn't there be a RN condition so we don't' do on web?